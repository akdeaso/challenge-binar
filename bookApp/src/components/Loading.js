import { StyleSheet, Text, View, ActivityIndicator } from 'react-native'
import React from 'react'
import { ms } from 'react-native-size-matters'

const Loading = () => {
    return (
        <View flex={1} justifyContent="center" alignItems="center" style={{ backgroundColor: "#191932" }} >
            <ActivityIndicator size={ms(50)} color={"white"} />
        </View>
    )
}

export default Loading

const styles = StyleSheet.create({})