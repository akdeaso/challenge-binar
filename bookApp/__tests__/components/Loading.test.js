import React from 'react';
import {create} from 'react-test-renderer';
import Loading from '../../src/components/Loading';

const component = create(<Loading />);

describe('should render loading', () => {
  test('matching snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
