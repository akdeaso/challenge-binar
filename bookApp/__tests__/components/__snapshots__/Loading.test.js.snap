// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading matching snapshot 1`] = `
<View
  alignItems="center"
  flex={1}
  justifyContent="center"
  style={
    Object {
      "backgroundColor": "#191932",
    }
  }
>
  <ActivityIndicator
    color="white"
    size={78.57142857142857}
  />
</View>
`;
